dbms/src/AggregateFunctions/AggregateFunctionUniqCombined.h

```diff
@@ -76,7 +81,7 @@ struct AggregateFunctionUniqCombinedDataWithKey
     // We want to migrate from |HashSet| to |HyperLogLogCounter| when the sizes in memory become almost equal.
     // The size per element in |HashSet| is sizeof(Key)*2 bytes, and the overall size of |HyperLogLogCounter| is 2^K * 6 bits.
     // For Key=UInt32 we can calculate: 2^X * 4 * 2 ≤ 2^(K-3) * 6 ⇒ X ≤ K-4.
-    using Set = CombinedCardinalityEstimator<Key, HashSet<Key, TrivialHash, HashTableGrower<>>, 16, K - 4, K, TrivialHash, Key>;
+    using Set = CombinedCardinalityEstimator<Key, HashSet<Key, TrivialHash, UniqCombinedHashTableGrower>, 16, K - 4 + (sizeof(Key) == sizeof(UInt32)), K, TrivialHash, Key>;
```

Comment:
But here we keep as before for UInt64 and increased by one for UInt32. That is inconsistent with uniqCombined(17).

Reply:
It means that we also need to add a test for `uniqCombined(16)(...)` or `uniqCombined(18)(...)`.

Reply:
Nice catch! (will fix this a little bit later)